src/recordpage.cpp

```diff
@@ -480,6 +488,9 @@ void RecordPage::onStartRecordingButtonClicked(bool checked)
         ui->startRecordingButton->setText(tr("Stop Recording"));
         ui->perfResultsTextEdit->clear();
 
+        // clear konsole
```

Comment:
same should be done in `updateRecordType` below where `ui->perfResultsTextEdit->clear();` is also called